client/a8c-for-agencies/sections/sites/features/jetpack/jetpack-sites-dataviews.tsx

```diff
@@ -0,0 +1,401 @@
+							tourId="sitesWalkthrough"
+							context={ scanRef }
+						/>
+					</>
+				),
+				getValue: () => '-',
+				render: ( { item }: { item: SiteInfo } ) => renderField( 'scan', item ),
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'plugins',
+				header: (
+					<>
+						<span
+							className="sites-dataview__plugins-header"
+							ref={ ( ref ) => setPluginsRef( ref as HTMLElement | null ) }
+						>
+							PLUGINS
+						</span>
+						<GuidedTourStep
+							id="sites-walkthrough-plugins"
+							tourId="sitesWalkthrough"
+							context={ pluginsRef }
+						/>
+					</>
+				),
+				getValue: () => '-',
+				render: ( { item }: { item: SiteInfo } ) => renderField( 'plugin', item ),
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'favorite',
+				header: (
+					<Icon
+						className="site-table__favorite-icon sites-dataview__favorites-header"
+						size={ 24 }
+						icon={ starFilled }
+					/>
+				),
+				getValue: ( { item }: { item: SiteInfo } ) => item.isFavorite,
+				render: ( { item }: { item: SiteInfo } ) => {
+					if ( isLoading ) {
+						return <TextPlaceholder />;
+					}
+					return (
+						<span className="sites-dataviews__favorite-btn-wrapper">
+							<SiteSetFavorite
+								isFavorite={ item.isFavorite || false }
+								siteId={ item.site.value.blog_id }
+								siteUrl={ item.site.value.url }
+							/>
+						</span>
+					);
+				},
+				enableHiding: false,
+				enableSorting: false,
+			},
+			{
+				id: 'actions',
+				getValue: ( { item }: { item: SiteInfo } ) => item.isFavorite,
+				render: ( { item }: { item: SiteInfo } ) => {
+					if ( isLoading ) {
+						return <TextPlaceholder />;
+					}
+					return (
+						<div className="sites-dataviews__actions">
+							<SiteActions
+								isLargeScreen={ isLargeScreen }
+								site={ item.site }
+								siteError={ item.site.error }
+							/>
+							<Button
+								onClick={ () => openSitePreviewPane( item.site.value ) }
+								className="site-preview__open"
+								borderless
+								ref={ ( ref ) => setActionsRef( ( current ) => current || ref ) }
+							>
+								<Gridicon icon="chevron-right" />
+							</Button>
+						</div>
+					);
+				},
+				header: (
+					<GuidedTourStep
+						id="sites-walkthrough-site-preview"
+						tourId="sitesWalkthrough"
+						context={ actionsRef }
+					/>
+				),
+				enableHiding: false,
+				enableSorting: false,
+			},
+		],
+		[
+			translate,
+			introRef,
+			statsRef,
+			boostRef,
+			backupRef,
+			monitorRef,
+			scanRef,
+			pluginsRef,
+			actionsRef,
+			isLoading,
+			openSitePreviewPane,
+			renderField,
+			isLargeScreen,
+		]
+	);
+
+	const urlParams = new URLSearchParams( window.location.search );
+	const isOnboardingTourActive = urlParams.get( 'tour' ) !== null;
+	const useExampleDataForTour =
+		forceTourExampleSite || ( isOnboardingTourActive && ( ! sites || sites.length === 0 ) );
```

Comment:
This is part of the Onboarding Tour code. This potentially could be moved to an upper layer, allowing any feature family to have their own Onboarding Tour.